Subject: do not link spice with libcacard
From: Michael Tokarev <mjt@tls.msk.ru>
Submitted-Upstream: yes

None of the libcacard symbols are actually used by the library,
only the header files, so there's no need to link with libcacard.

Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>

diff --git a/spice-common/m4/spice-deps.m4 b/spice-common/m4/spice-deps.m4
index e4b2c8d..d1c39d0 100644
--- a/spice-common/m4/spice-deps.m4
+++ b/spice-common/m4/spice-deps.m4
@@ -58,7 +58,7 @@ AC_DEFUN([SPICE_CHECK_SMARTCARD], [
       fi
     fi
     AS_VAR_APPEND([$1_CFLAGS], [" $SMARTCARD_CFLAGS"])
-    AS_VAR_APPEND([$1_LIBS], [" $SMARTCARD_LIBS"])
+dnl   AS_VAR_APPEND([$1_LIBS], [" $SMARTCARD_LIBS"])
 ])
 
 
